### DISCUSSION TITLE
Error:Cannot download 'https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css': Remote host terminated the handshake

### DISCUSSION THREAD

**Discussion by mina20088** [twbs/bootstrap#38732] [General]
hi i have a problem when try to download the bootstrap from php storm when i right clicked on the link to download the lib i got this error 
![Screenshot 2023-06-08 224408](https://github.com/twbs/bootstrap/assets/39167278/66affb7f-deee-4b5f-8be4-4e020c19697e)

**Comment by julien-deramond** [comment]
Hi @mina20088 
You are probably behind a corporate proxy. So I'd say you need to [configure PHP Storm to set up the HTTP proxy settings](https://www.jetbrains.com/help/phpstorm/settings-http-proxy.html).
You can try to open https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js directly in your browser in the same environment, and check what is the network configuration of your browser to duplicate it in PHP Storm.

**Reply by mina20088** [reply]
Hi @julien-deramond  
this is a private laptop and i work as a free lancer so i am working on my personal network not belong to a company

**Reply by gerardnico** [reply]
@julien-deramond gave you a pretty good answer.

There is a ssl handshake problem. We can't see the whole error.

Can you download the library from the browser?

Try to see if the request arrive to the external server with tracing
https://datacadamia.com/network/route#tracing

You can also try to restart php storm. It may read the network configuration at startup... who knows.

Best of luck

**Reply by mina20088** [reply]
i can send you the error
[idea.log](https://github.com/twbs/bootstrap/files/11716430/idea.log)

**Reply by mina20088** [reply]
this file contains the error

**Reply by mina20088** [reply]
@gerardnico  it is not working even when accessed from the browser this link is not working in the browser there is a problem in the server from your side check the server
https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css

**Reply by julien-deramond** [reply]
I can access this link as well. Maybe it's blocked by something else 🤷 
You can try another provider to see if it's better for you (e.g.: https://unpkg.com/bootstrap@5.3.0/dist/css/bootstrap.min.css)

**Reply by mina20088** [reply]
@julien-deramond  i got this error in the inspector of the firefox when i but the cdns
![Screenshot from 2023-06-14 01-33-50](https://github.com/twbs/bootstrap/assets/39167278/806711dc-1627-47bc-ab88-c110fd8a6432)

**Reply by julien-deramond** [reply]
With the complete error, it's clearer indeed.

> None of the "sha384" hashes in the integrity attribute match the content of the subresource

When you integrate the CDN versions, be careful with the integrity value that changes for each version (see https://getbootstrap.com/docs/5.3/getting-started/introduction/#quick-start)

```html
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/js/bootstrap.bundle.min.js" integrity="sha384-geWF76RCwLtnZ8qwWowPQNguL3RmwHVBC9FhGdlKrxdiJJigb/j/68SIy3Te4Bkz" crossorigin="anonymous"></script>
```
